app/src/main/java/com/es/findsoccerplayers/Utils.java

```diff
@@ -88,4 +89,110 @@ public void onSuccess(Void aVoid) {
     public static void showCannotSendMessage(Context c){
         Toast.makeText(c, R.string.send_empty_message, Toast.LENGTH_SHORT).show();
     }
+
+    public static String getPreviewDescription(String description){
+        if(description.length()<20)
+            return description;
+        else{
+            return (description.substring(0,21)+"...");
+        }
+    }
+

```

Comment:
perché hai scritto questi due metodi? Usa `Calendar.getDisplayName`

Reply:
Non mi pare faccia quello che voglio quel metodo

Reply:
Calendar non va bene. Ma potresti usare SimpleDateFormat di Java che ti da anche i nomi nella lingua desiderata
https://www.mrwebmaster.it/java/simpledateformat-formattare-data-java_11616.html

Reply:
si fa esattamente questo, ti estrae i field: https://docs.oracle.com/javase/7/docs/api/java/util/Calendar.html#getDisplayName(int,%20int,%20java.util.Locale)

Reply:
puoi anche scegliere se visualizzare in formato lungo o corto

Reply:
sinceramente non ho ben capito come implementarlo, magari potete farlo voi se avete già un'idea di come fare